website/docs/basics/blocks-styles.md

```diff
@@ -245,6 +245,57 @@ Variable `value` will output all variables depending on attributes value. Everyt
 --typography-line-height: 144px;
 ```
 
+## Variables Editor 
+
+Editor variables behave exactly the same as `variables`, except they will be printed out only in editor part.
+They are mostly used for overriding of the actual behavior inside the admin for the purpose of better control of the content.
+Example is based on common case where you don't want to remove element from the editor part but still somehow indicate that it will be hidden.
```

Comment:
```suggestion
The following example is based on a common case where you don't want to remove the element from the editor part but still somehow indicate that it will be hidden.
```